### DISCUSSION TITLE
How to implement svgs for symbology that exist outside of QGIS svg library

### DISCUSSION THREAD

**Discussion by Jdgilby** [Riverscapes/QRiS#303] [Ideas & Feature Requests]
@joewheaton @philipbaileynar @KellyMWhitehead [Here's ](https://youtu.be/cOqyY_lWwEk)a video of what im talking about.

**Comment by joewheaton** [comment]
@Jdgilby good questions... I think there are at least three questions here. So clearly you can in a local map session load a symbol using a local SVG. The work around to all in the meantime is to just choose a symbol from what QGIS ships with. Below are the questions (I don't know answers but I speculate) and who might be able to help...
1. How to get MUI icons as SVGs? @philipbaileynar, I realized I don't know how to do this. Do you know?
2. Whether a QML can point to a SVG and where it has to be (e.g. local or relative to location of SVG... in our case Viewer/Resources/Symbology/QGIS/QRiS) - @KellyMWhitehead?
3. If we have to ship the SVG as a file, does it belong in RiverscapesXML symbology?

@philipbaileynar if we are barking up the wrong tree here, just set us straight. Thanks.

**Comment by philipbaileynar** [comment]
I agree with Joe, stick with built in symbols for now. There are 50 billion to choose from!

Specifically, I have been getting the MUI icons as SVG or PNG from [here](https://fonts.google.com/icons). (Note these are not actually the latest MUI icons but they are veeeeeeeery close. And Google keeps changing them, so perfection is not possible.)

Unless QGIS allows for symbols at URLs, I think the workarounds to make this happen (SVGs in RiverscapesXML that get synced locally and then intercepting symbols during layer load to point to sync folder) is a crap ton of work that is hard to prioritize above other QRiS efforts.

**Reply by joewheaton** [reply]
Super helpful @philipbaileynar. @Jdgilby and @nick4rivers, on a rainy day, if we decide to customize some of this for Anabranch, that's something finicky we can do. But for now, lets stick with the built in ones.

**Comment by joewheaton** [comment]
@philipbaileynar.... It just ocurred to me. your comment: 
> Unless QGIS allows for symbols at URLs....

I was not thinking ... That is exactly what https://plugins.qgis.org/styles/?order_by=-upload_date&&is_gallery=true is. Just like contributing a Plugin, you can contribute symbology. Then that symbology is explicitly both available by URL and through the normal channels in Q. What I don't know is if we can control, which of those custom symbols can get used if a user hasn't turned them on, but we _could_ explore that? Am I off base @philipbaileynar?
